### PR TITLE
feat: Adds a new validator to deal with specified types of  extensions

### PR DESCRIPTION
**Pull request by matheusnascgomes**
- Creates the new validator function
- Creates tests of this new validator function
- Updates validators section on README

Signed-off-by: Matheus Gomes <matheus.gomes@justa.com.vc>

### CODE REVIEW

File: libs/material-file-input/src/lib/validator/file-validator.ts [hunk 1]
```diff
@@ -23,4 +23,32 @@ export namespace FileValidator {
...
+   */
+  export function allowedExtensions(extentions: string[]): ValidatorFn {
+    return (control: AbstractControl): { [key: string]: any } | null => {
+      const allowed =
+        control &&
+        control.value &&
+        (control.value as FileInput).files.every(file => {
+          const extention = file.name.split('.')[1];
+
+          const foundIndex = extentions.indexOf(extention);
```
Comment (raisiqueira) [inline libs/material-file-input/src/lib/validator/file-validator.ts]: lastIndexOf it’s a good approach to this case. (I.e: file.name.pdf)

**Comment by matheusnascgomes** [comment]
Hi @merlosy 
Do you have any interesting on merge this new features?

**Comment by raisiqueira** [comment]
Hey @matheusnascgomes can you publish into your NPM account and give the credits to @merlosy. 4 months with a great improvement and still open :(

**Comment by matheusnascgomes** [comment]
> Hey @matheusnascgomes can you publish into your NPM account and give the credits to @merlosy. 4 months with a great improvement and still open :(

Fair enough, i'll consider it!

**Pull request closed without merging**